src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -77,6 +77,11 @@
             short-summary: Update an availability set
             """
 
+helps['vm availability-set convert'] = """
+            type: command
+            short-summary: convert an availability set to contain VMs with managed disks
```

Comment:
This actually changes the meaning. This `convert` only flips a flag to be `managed disk aware`, and nothing happens on individual VMs inside, and users need to run a different convert on each individual VM subsequently